### PR TITLE
✨ Add riders page

### LINKED ISSUE #130: Add Riders Page

### PR DESCRIPTION
**Pull request by arassaria**
# Description

Fixes #130 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -8,13 +8,17 @@ import Tours from "./pages/Tours";
...
+import Rider from "./pages/Rider";
 
 function App() {
   return (
     <>
       <GlobalStyle />
       <Router>
         <Switch>
+          <Route path="/riders">
+            <Rider />
```
Comment (lmachens) [inline client/src/App.js]: If the route is `riders`, I expect multiple riders in the page.

Rename the page to `Riders` or the route to `/rider` :)

File: client/src/pages/Rider.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  const [riders, setRiders] = useState([]);
+  const history = useHistory();
+
+  useEffect(() => {
+    const doFetch = async () => {
+      const data = await getSortedData({
+        collectionName: "riders",
+        dataName: "alias",
+      });
+      console.log({ data });
```
Comment (lmachens) [inline client/src/pages/Rider.js]: Logs like these are great in development, but should not land in production.

File: client/src/pages/Rider.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        {riders?.map((rider) => (
+          <CardRider
+            key={rider._id}
+            name={rider.name}
+            alias={rider.alias}
+            dateOfBirth={rider.dateOfBirth}
+            phone={rider.phone}
+            picture={rider.picture}
+            color={rider.color}
+          />
```
Comment (lmachens) [inline client/src/pages/Rider.js]:
```suggestion
          <CardRider
            key={rider._id}
            {...rider}
          />
```

**Review by lmachens — COMMENTED** [review]
Good job :)

File: client/src/pages/Rider.js (hunk 3 above)
Comment (arassaria) [reply client/src/pages/Rider.js]: Thanks, was sure there was an easier way, but hadn't looked for it yet. 😄

File: client/src/pages/Rider.js (hunk 2 above)
Comment (arassaria) [reply client/src/pages/Rider.js]: Commented on that on someone elses pull request and forgot to remove it myself. 😅

File: client/src/App.js (hunk 1 above)
Comment (arassaria) [reply client/src/App.js]: Resolved 😄

**Review by bwnstck — APPROVED** [review]
really nice one 🦸‍♂️

**Pull request merged**